config/StoredProcedures/CreateUser.sql

```diff
@@ -15,7 +15,7 @@ union
 select GroupName as name from UserGroups) as s1 where s1.name=_Username;
 
 IF @unique<=0 THEN
-	INSERT INTO Users (Username, FirstName, LastName, Email, PermsID, HP, US, DateCreated)  VALUES(_Username,_FirstName,_LastName,_Email,@pmid,_HP,_US,_DateCreated);
+	INSERT INTO Users (Username, FirstName, LastName, Email, PermsID, HP, US, DateCreated)  VALUES(_Username,_FirstName,_LastName,_Email,@pmid,_HP,_US,_DateCreated, (1));
```

Comment:
Then we should add the isConfirmed column name as well. @mrstarware: the column names are the things between "Users" and "Values", and they specify which column each argument should go into. That way if you don't know the order they're in on the actual table, you can just name all the columns in the order you'd like to enter the data.